services/src/main/java/org/keycloak/broker/saml/mappers/AttributeToRoleMapper.java

```diff
@@ -17,6 +17,7 @@
 
 package org.keycloak.broker.saml.mappers;
 
+import org.jboss.logging.processor.util.Objects;
```

Comment:
Please don't use Objects from org.jboss.logging. The original comment was referring to Objects.equals from standard java.util package